frontend/packages/dev-console/src/components/gitops/GitOpsListPage.scss

```diff
@@ -0,0 +1,10 @@
+.odc-gitops-list-page-heading {
+    align-items: center;
+    display: inline-block;
+    font-weight: normal;
+    font-size: 12px;
```

Comment:
You probably don't need these styles if you use `<Split>` component along with `<SplitItem>`